invideoquiz/utils.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"""
+Make '_' a no-op so we can scrape strings
+"""
+
+
+def _(text):
```

Comment:
It's not clear to me how the code that calls this will know to use the `gettext` version instead of this,
when it actually needs to pull the live translations during production.